pipeline-model-definition/src/test/resources/options/restartableFromStageDisabled.groovy

```diff
@@ -0,0 +1,40 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, CloudBees, Inc.
```

Comment:
Probably best to just delete the license headers here. IDK why we include them in existing test resources.

Reply:
Please verify the change